package.json

```diff
@@ -1,17 +1,34 @@
 {
   "name": "eslint-config-zipscene",
-  "version": "0.0.6",
+  "version": "0.1.0",
```

Comment:
It was because of the change to the paths of the files, frontend/backend. Which I just added back. So I'll drop the version again :).